runtime/manifest.js

```diff
@@ -61,7 +61,7 @@ class Manifest {
       view = new Variable(type, this, name, id);
     }
     this._views.push(view);
-    this._viewTags.set(view, tags);
+    this._viewTags.set(view, tags ? tags : []);
```

Comment:
i'm surprised that it would be needed because right before calling the newView(...) method
https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest.js#L450
we do 
    if (tags == null)
      tags = [];
https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest.js#L437
did it really solve the problem you were having?

anyway, i would think the [] default value might be better here:
https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest-parser.peg#L56

Reply:
I don't have the repro handy (although I could get back to it), but I think the caller is way over at https://github.com/PolymerLabs/arcs-cdn/blob/gh-pages/dev/app-shell/lib/metadata-storage.js#L95.

Reply:
Ohk, i see now that it is being called from outside the arcs code. Thanks for the clarification!

Reply:
Yeah! And it's a good question, I definitely went back-and-forth on where to put the check.